### DISCUSSION TITLE
v1.4.1

### DISCUSSION THREAD

**Discussion by ricoberger** [feeddeck/feeddeck#187] [Announcements]
### Fixed

- #186: [core] Fix Warnings and Regenerate Splash Screens @ricoberger
- #185: [core] Fix Web Build @ricoberger
- #184: [core] Fix Metadata for Flatpak @ricoberger
- #183: [core] Upload Raw Build in Windows CI/CD Pipeline @ricoberger
- #182: [podcast] Fix Audio Playback on Windows and Linux @ricoberger

<hr /><em>This discussion was created from the release <a href='https://github.com/feeddeck/feeddeck/releases/tag/v1.4.1'>v1.4.1</a>.</em>

**Comment by bobster151** [comment]
Hi, my feeds haven't updated for a month, hitting refresh does nothing, anything I can do?

**Reply by ricoberger** [reply]
Hi are you using the hosted version or do you host it by yourself?

If you are using the hosted version:

- Do you have an active subscription, because the free version only works for some days to try it out
- If you have an active subscription can send me an email to admin@feeddeck.app from the email address you used for the registration 

If you host it by yourself:

- Please check if the feeds are related to a source which was removed in the last update e.g. Nitter 🙁
- Check the logs of the Docker container which is responsible for updating the feeds

**Comment by bobster151** [comment]
Hi,

I can see now I misread the website andI have to go premium if I want to
continue after 7 days.

Thanks,

Bob
